app/boards/shields/microdox/microdox.keymap

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/keys.h>
+#include <dt-bindings/zmk/bt.h>
+
+/ {
+        keymap {
+                compatible = "zmk,keymap";
+
+                default_layer {
+// -----------------------------------------------------------------------------------------
+// |  Q  |  W  |  E  |  R  |  T   |   |  Y  |  U   |  I  |  O  |  P  |
+// |  A  |  S  |  D  |  F  |  G   |   |  H  |  J   |  K  |  L  |  ;  |
+// |  Z  |  X  |  C  |  V  |  B   |   |  N  |  M   |  ,  |  .  |  /  |
+//             | GUI | NAV | SHFT |   | SPC | SYM  | ALT |
+                        bindings = <
+   &kp Q &kp W &kp E &kp R &kp T   &kp Y &kp U  &kp I    &kp O   &kp P
+   &kp A &kp S &kp D &kp F &kp G   &kp H &kp J  &kp K    &kp L   &kp SCLN
+   &kp Z &kp X &kp C &kp V &kp B   &kp N &kp M  &kp CMMA &kp DOT &kp FSLH
+          &kp LGUI &mo 1 &kp LSFT   &kp SPC &mo 2 &kp RALT
+                        >;
+                };
+                nav_layer {
+// -----------------------------------------------------------------------------------------
+// |     |     | ESC |  ~   |     |   | TAB | HOME |  UP  | END | DEL  |
+// |     | GUI | ALT | CTRL | NUM |   |  /  | LEFT | DOWN | RGT | BKSP |
+// |     |     |     |      |     |   |  \  | ENT  |      |     |      |
+//             |     |      |     |   |     |      |      |
+                        bindings = <
+   &trans &trans  &kp ESC  &kp TILD &trans     &kp TAB  &kp HOME &kp UARW &kp END  &kp DEL
+   &trans &kp GUI &kp RALT &kp LCTL &mo 3      &kp FSLH &kp LARW &kp DARW &kp RARW &kp BKSP
+   &trans &trans  &trans   &trans   &trans     &kp BSLH &kp RET  &trans   &trans    &trans
+                  &trans   &trans   &trans     &trans   &trans   &trans
+                        >;
+                };
+
+                sym_layer {
+// -----------------------------------------------------------------------------------------
+// |  !  |  @  |  #  |  $  |  %  |   |  ^  |  &  |  *  |  (  |  )  |
+// |     |     |     |     |     |   |  -  |  =  |  {  |  }  | "|" |
+// |     |     |     |     |     |   |  _  |  +  |  [  |  ]  |  \  |
+//             | GUI |     | SPC |   | ENT |     | ALT |
+                        bindings = <
+   &kp BANG  &kp ATSN  &kp HASH  &kp CURU  &kp PRCT    &kp CRRT  &kp AMPS &kp KMLT &kp LPRN &kp RPRN
+   &trans    &trans    &trans    &trans    &trans      &kp MINUS &kp EQL  &kp LBKT &kp RBKT &kp PIPE
+   &trans    &trans    &trans    &trans    &trans      &trans    &trans   &trans   &trans   &kp BSLH
+                       &kp LGUI  &trans    &kp SPC     &kp RET   &trans   &kp RALT
+                        >;
+                };
+
+                num_layer {
```

Comment:
Until we figure out a sane way to do "tri layer state", this layer isn't reachable. If you want to leave it in here, I don't have a problem with it, but maybe a comment to that effect here would be good so users know.